src/main/java/com/ounitech/wemove/services/MemberService.java

```diff
@@ -74,7 +74,7 @@ public List<Member> findByInactiveMembers() {
         return memberRepository.findByActive(false);
     }
 
-    public Member findByEmail(String email) {
+    public Optional<Member> findByEmail(String email) {
         return memberRepository.findByEmail(email);
```

Comment:
no need to propagate the signature in the service layer
we can keep the old signature and return null if no Member found : this will be enhanced later